packages/core/src/getPathFromState.tsx

```diff
@@ -59,15 +62,37 @@ const getActiveRoute = (state: State): { name: string; params?: object } => {
  * @returns Path representing the state, e.g. /foo/bar?count=42.
  */
 export default function getPathFromState(
-  state?: State,
-  options: PathConfig = {}
+  state: State,
+  options?: Options
 ): string {
-  if (state === undefined) {
-    throw Error('NavigationState not passed');
+  if (state == null) {
+    throw Error(
+      "Got 'undefined' for the navigation state. You must pass a valid state object."
```

Comment:
Is it `undefined` or `null` then?

Reply:
Yea, the types prevent it anyway so shouldn't be an issue